jsonschema2pojo-maven-plugin/src/main/java/org/jsonschema2pojo/maven/Jsonschema2PojoMojo.java

```diff
@@ -394,364 +375,337 @@ public class Jsonschema2PojoMojo extends AbstractMojo implements GenerationConfi
 
     /**
      * Whether to make the generated types 'serializable'.
      *
-     * @parameter property="jsonschema2pojo.serializable" default="fals"
      * @since 0.4.23
      */
+    @Parameter(property = "jsonschema2pojo.serializable", defaultValue = "false")
     private boolean serializable = false;
 
     /**
      * Whether to initialize Set and List fields as empty collections, or leave
      * them as <code>null</code>.
      *
-     * @parameter property="jsonschema2pojo.initializeCollections"
-     *            default-value="true"
      * @since
      */
+    @Parameter(property = "jsonschema2pojo.initializeCollections", defaultValue = "true")
     private boolean initializeCollections = true;
 
     /**
      * List of file patterns to include.
      *
-     * @parameter
      * @since 0.4.3
      */
+    @Parameter
     private String[] includes;
 
     /**
      * List of file patterns to exclude. This only applies to the initial scan
      * of the file system and will not prevent inclusion through a "$ref" in one
      * of the schemas.
      *
-     * @parameter
      * @since 0.4.3
      */
+    @Parameter
     private String[] excludes;
 
     /**
      * Whether to add a prefix to generated classes.
      *
-     * @parameter property="jsonschema2pojo.classNamePrefix"
      * @since 0.4.6
      */
+    @Parameter(property = "jsonschema2pojo.classNamePrefix")
     private String classNamePrefix = "";
 
     /**
      * Whether to add a suffix to generated classes.
      *
-     * @parameter property="jsonschema2pojo.classNameSuffix"
      * @since 0.4.6
      */
+    @Parameter(property = "jsonschema2pojo.classNameSuffix")
     private String classNameSuffix = "";
 
     /**
      * The strings (no preceeding dot) that should be considered as file name
      * extensions, and therefore ignored, when creating Java class names.
      *
-     * @parameter property="jsonschema2pojo.fileExtensions"
-     *            default-value=""
      * @since 0.4.23
      */
+    @Parameter(property = "jsonschema2pojo.fileExtensions", defaultValue = "")
     private String[] fileExtensions = new String[] {};
 
     /**
      * Whether to generate constructors or not
      *
-     * @parameter property="jsonschema2pojo.includeConstructors"
-     *            default-value="false"
      * @since 0.4.8
      */
+    @Parameter(property = "jsonschema2pojo.includeConstructors", defaultValue = "false")
     private boolean includeConstructors = false;
 
     /**
-     * The 'constructorsRequiredPropertiesOnly' configuration option. This is a legacy configuration option used to turn on the {@link
-     * #isIncludeAllPropertiesConstructor()} and off the {@link * #isConstructorsIncludeAllPropertiesConstructor()} configuration options.
-     * It is specifically tied to the {@link #isIncludeConstructors()} * property, and will do nothing if that property is not enabled
+     * The 'constructorsRequiredPropertiesOnly' configuration option.
```

Comment:
Since given Javadoc contained invalid `{@link ...}` value as well as inaccurate text it had to be rewritten a bit.
Needs to be checked whether description is better now or needs changes